Tests/Metadata/Driver/Fixture/Service.php

```diff
@@ -11,7 +11,9 @@
  *     decorates="test.service",
  *     decoration_inner_name="original.test.service",
  *     deprecated="use new.test.service instead",
- *     public=false
+ *     public=false,
+ *     autowire=false,
+ *     autowiring_types={"JMS\DiExtraBundle\Tests\Metadata\Driver\Fixture\Service"},
```

Comment:
`Service::class` makes the travis run for PHP 5.3 + prefer-lowest fail...
However PHP 5.3 with standard dependencies pass...

Still, I guess I will revert to the literal. (Unless you know what (dev) dependency I should update to make it pass?)